povm_toolbox/sampler/povm_sampler_job.py

```diff
@@ -88,6 +90,83 @@ def result(self) -> PrimitiveResult[POVMPubResult]:
 
```

Comment:
I just noticed that the docstring of this class's `__init__` contains a wrong argument: `povm` which should be removed.